src/utils/vaultMath.ts

```diff
@@ -174,11 +184,12 @@ export const collateralizationRatio = (
   price: PriceDescription,
```

Comment:
now that there are two price params, they should be distinguished. e.g. `marketPrice`

Reply:
How about `quotePrice`? Because it's under `.quote` in vstorage.